src/sentry/static/sentry/app/views/app.jsx

```diff
@@ -143,7 +143,7 @@ const App = createReactClass({
     if (Object.keys(newState).length > 0) this.setState(newState);
   },
 
-  @keydown('cmd+shift+p')
+  @keydown('meta+shift+p', 'meta+k')
```

Comment:
trying this, less interference with chrome dev tools (its slack's switcher hot key)

Reply:
hmmm, I like it.

Reply:
what's the difference between cmd and meta? 

docs weren't 100% clear to me
>`meta` (aka `command` or `cmd`)